src/base/bittorrent/session.cpp

```diff
@@ -2826,13 +2905,8 @@ void Session::setMaxConnectionsPerTorrent(int max)
         m_maxConnectionsPerTorrent = max;
 
         // Apply this to all session torrents
-        for (const lt::torrent_handle &handle : m_nativeSession->get_torrents()) {
-            if (!handle.is_valid()) continue;
-            try {
-                handle.set_max_connections(max);
-            }
-            catch (const std::exception &) {}
-        }
+        for (const TorrentHandle *torrent : asConst(torrents()))
+            torrent->setMaxConnections(max);
```

Comment:
Are you sure this action (and two following) shouldn't affect "hidden" torrents?

Reply:
What do you mean by "hidden" torrents?

Reply:
I should revert these.